README.md

```diff
@@ -34,13 +34,46 @@ You can automatically generate a clover report file with every PHPUnit run by ad
     </coverage>
 ```
 
-The default configuration does **NOT** push the badge back into the repository. 
-Setting the input **push_badge** in your workflow to true will enable the commit and push process and the badge will be pushed into the repository after it's been generated. In addition, pushing the badge requires you to update your project settings under `Code and automation` > `Actions` > `General` > `Workflow permissions` to `Read and write permissions`.
+# Default configuration
+
+The default configuration commits the output badge to branch `image-data`. As a consequence, this avoids the list of issues with the alternate configuration (see below), namely:
```

Comment:
Same as in the heading.

Reply:
Maybe you could also mention that it the config uses a separate action, "peaceiris/actions-gh-pages@v3", to push the badge?

Reply:
done